src/Aspire.Hosting.Azure.Storage/AzureStorageResource.cs

```diff
@@ -63,12 +66,21 @@ void IResourceWithAzureFunctionsConfig.ApplyAzureFunctionsConfiguration(IDiction
     {
         if (IsEmulator)
         {
-            target[connectionName] = GetEmulatorConnectionString();
+            // Injected to support Azure Functions listener initialization.
+            var connectionString = GetEmulatorConnectionString();
+            target[connectionName] = connectionString;
+            // Injected to support Aspire client integration for Azure Storage.
+            target[$"{BlobsConnectionKeyPrefix}__{connectionName}__ConnectionString"] = connectionString;
+            target[$"{QueuesConnectionKeyPrefix}__{connectionName}__ConnectionString"] = connectionString;
```

Comment:
why aren't we doing tables?

Reply:
I don't think there's a specific reason other than the fact that it's not in the original list of resource types we prioritized for the integration (Azure Storage Queues/Blobs, Service Bus, EventHubs).